src/test/regress/expected/multi_metadata_sync.out

```diff
@@ -1812,6 +1836,55 @@ SELECT run_command_on_workers('GRANT ALL ON pg_dist_local_group TO non_super_met
  (localhost,57638,t,GRANT)
 (2 rows)
 
+SELECT run_command_on_workers('GRANT ALL ON SCHEMA citus TO non_super_metadata_user');
+  run_command_on_workers
+---------------------------------------------------------------------
+ (localhost,57637,t,GRANT)
+ (localhost,57638,t,GRANT)
+(2 rows)
+
+SELECT run_command_on_workers('ALTER SEQUENCE user_defined_seq OWNER TO non_super_metadata_user');
```

Comment:
Hmm, this change seems suspicious. Why do we need to change the ownership of the sequence?  I mean, we don't change the ownership of the relevant table but only the sequence?